infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/external/TraefikGatewayRouteConfigGeneratorTest.java

```diff
@@ -30,10 +30,13 @@
 public class TraefikGatewayRouteConfigGeneratorTest {
 
   private GatewayRouteConfigGenerator gatewayConfigGenerator;
+  private GatewayRouteConfigGenerator gatewayConfigGeneratorNonDefaultDomain;
 
   @BeforeMethod
   public void setUp() {
-    gatewayConfigGenerator = new TraefikGatewayRouteConfigGenerator();
+    gatewayConfigGenerator = new TraefikGatewayRouteConfigGenerator("cluster.local");
```

Comment:
@tolusha it will be null or empty string?

Reply:
I believe it will be empty string